docs/synthetics/ci-cd/ci-cd-installation.md

```diff
@@ -0,0 +1,283 @@
+          TARGET_URL: ${{ env.TARGET_URL }}                   # The URL of the deployment which you want to test
+          USE_HEAD_SHA: false                                 # Set to true to use the HEAD SHA for the check run instead of GIT_COMMIT_HASH
+
+      - name: Update Job Status
+        id: update_status
+        if: always() && steps.create_check.outcome == 'success'
+        run: |
+          JOB_DETAILS="---- Job Details -----<br>"
+          
+          # Optionally add a URL to the PR for which these tests are being run, if you passed that info in the invoking event
+          #PR_URL="${{ github.server_url }}/${{ github.repository }}/pull/$PR_NUMBER"
+          #JOB_DETAILS="$JOB_DETAILS- **GitHub PR '${{ env.SOURCE }}' is available** <a href='$PR_URL' target='_blank'>here</a><br>"
+
+          RUN_URL="${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+          GROUP_RUN_URL="${{ steps.sematext_check.outputs.group_run_url }}"
+          JOB_DETAILS="$JOB_DETAILS- **GitHub Workflow Run is available** <a href='$RUN_URL' target='_blank'>here</a><br>"
+          if [ -n "$GROUP_RUN_URL" ]; then
+            JOB_DETAILS="$JOB_DETAILS- **Sematext Group Run is available** <a href='$GROUP_RUN_URL' target='_blank'>here</a><br>"
+          fi
+          JOB_DETAILS="$JOB_DETAILS- **Testing finished at:** $(date)<br>"
+          JOB_DETAILS="$JOB_DETAILS- **Commit:** <a href='${{ github.server_url }}/${{ github.repository }}/commit/${{ env.SHA }}' target='_blank'>${{ env.SHA }}</a><br>"
+          JOB_DETAILS="$JOB_DETAILS- **TARGET_URL:** ${{ env.TARGET_URL }}<br><br>"
+          JOB_DETAILS="$JOB_DETAILS----- Test Results -----"
+
+          # Print out the invididual action outputs for debugging, if needed
+          echo "Sematext Check Status: '${{ steps.sematext_check.outputs.status }}'"
+          echo "Sematext Check Result: '${{ steps.sematext_check.outputs.result }}'"
+          echo "Sematext Check Error: '${{ steps.sematext_check.outputs.error }}'"
+
+          # Process the result to ensure proper formatting in GitHub Markdown
+          if [[ -n "${{ steps.sematext_check.outputs.result }}" ]]; then
+            FORMATTED_RESULT=$(echo '${{ steps.sematext_check.outputs.result }}' | sed 's/\\n/\n/g')
+            RESULTS_BLOCK="<pre>${FORMATTED_RESULT}</pre>"
+          else
+            RESULTS_BLOCK=""
+          fi
+
+          if [[ "${{ steps.sematext_check.outputs.status }}" == "passed" ]]; then
+            gh api "repos/${{ env.REPO }}/check-runs/${{ env.check_run_id }}" -X PATCH \
+              -F "name=Sematext CI/CD Test" \
+              -F "output[title]=Sematext Synthetics Check Passed" \
+              -F "output[summary]=All monitors finished successfully." \
+              -F "output[text]=$JOB_DETAILS<br>${RESULTS_BLOCK}" \
+              -F "status=completed" \
+              -F "conclusion=success"
+          else
+            FAILURE_REASON="Some monitors have failed."
```

Comment:
@HSSalman we use "monitors" and "checks" interchangeably. Won't be confusing?

Reply:
Hmm... right. We sort of have to use both because we're talking about GitHub Check Runs and our own Synthetics Monitors. I'll take another look to see if I can make things more consistent, but we'll still have both terms.